src/api/index.test.js

```diff
@@ -644,3 +644,66 @@ it('getResource', () => {
       fetchMock.restore();
     });
 });
+
+it('getTriggerTemplates', () => {
+  const data = {
+    items: 'triggerTemplates'
+  };
+  fetchMock.get(/triggertemplates/, data);
+  return index.getTriggerTemplates().then(triggerTemplates => {
+    expect(triggerTemplates).toEqual(data.items);
+    fetchMock.restore();
+  });
+});
+
+it('getTriggerTemplate', () => {
+  const name = 'foo';
+  const data = { fake: 'triggerTemplate' };
+  fetchMock.get(`end:${name}`, data);
+  return index.getTriggerTemplate({ name }).then(triggerTemplate => {
+    expect(triggerTemplate).toEqual(data);
+    fetchMock.restore();
+  });
+});
```

Comment:
super nit: you don't have to fix this, but the ordering of the TriggerTemplate stanzas is different from the others (the others are singular then plural) 😁

Reply:
True, done